source/db/en-projects.json

```diff
@@ -3918,6 +3918,117 @@
   },
   {
     "development_stage": "released",
+    "description": "A safe home for all your data",
+    "license_url": "https://github.com/nextcloud/server/blob/master/COPYING-README",
+    "logo": "nextcloud.png",
+    "notes": "Nextcloud provides a file sync & share solution that respects your privacy and is fully open source. Sync clients are available for all major platforms. Nextcloud comes with a rich ecosystem of apps including Contacts, Calendar, Video calls, Mail, Newsreader and much more.",
+    "privacy_url": "",
+    "source_url": "https://github.com/nextcloud",
+    "name": "Nextcloud",
+    "tos_url": "",
+    "url": "http://nextcloud.com",
+    "wikipedia_url": "https://en.wikipedia.org/wiki/Nextcloud",
+    "protocols": [
+      "CalDAV",
+      "CardDAV",
+      "IMAP",
+      "RSS",
+      "SSL/TLS",
+      "WebDAV",
+      "WebRTC",
+      "XMPP"
+    ],
+    "categories": [
+      {
+        "name": "GNU/Linux",
+        "subcategories": [
+          "Bookmark Sync",
+          "Email Clients",
+          "Enterprise Suite",
+          "File Storage & Sync",
+          "Media Publishing",
+          "Password Managers",
+          "Productivity",
+          "Video & Voice"
+        ]
+      },
+      {
+        "name": "BSD",
+        "subcategories": [
+          "Bookmark Sync",
+          "Email Clients",
+          "Enterprise Suite",
+          "File Storage & Sync",
+          "Media Publishing",
+          "Password Managers",
+          "Productivity",
+          "Video & Voice"
+        ]
+      },
+      {
+        "name": "Windows",
+        "subcategories": [
+          "Bookmark Sync",
+          "Email Clients",
+          "Enterprise Suite",
+          "File Storage & Sync",
+          "Media Publishing",
+          "Password Managers",
+          "Productivity",
+          "Video & Voice"
+        ]
+      },
+      {
+        "name": "OS X",
+        "subcategories": [
+          "Bookmark Sync",
+          "Email Clients",
+          "Enterprise Suite",
+          "File Storage & Sync",
+          "Media Publishing",
+          "Password Managers",
+          "Productivity",
+          "Video & Voice"
+        ]
+      },
+      {
+        "name": "Android",
+        "subcategories": [
+          "Enterprise Suite",
+          "File Storage & Sync",
+          "Media Publishing",
+          "Productivity",
+          "Video & Voice"
+        ]
+      },
+      {
+        "name": "iOS",
+        "subcategories": [
+          "Enterprise Suite",
+          "File Storage & Sync",
+          "Media Publishing",
+          "Productivity",
+          "Video & Voice"
+        ]
+      },
+      {
+        "name": "Servers",
```

Comment:
Probably all the GNU/Linux, BSD, etc. entries should be removed. This is consistent with other selfhosted server stuff we list (Piwik, Prosody, Postfix)